docs/operating-scylla/nodetool-commands/snapshot.rst

```diff
@@ -25,7 +25,9 @@ OPTIONS
 ====================================================================  =====================================================================================
 Parameter                                                             Description
 ====================================================================  =====================================================================================
--cf <table> / --column-family <table>, --table <table>                The table name (you must specify one and only one keyspace for using this option)
+-cf <table> / --column-family <table>, --table <table>                The table name (you must specify one and only one keyspace for using this option).
+                                                                      The table could be a regular table, a materialized view or index. In case of index
+                                                                      physical index name should be used.
```

Comment:
I'm not sure we use the phrase "physical index name" anywhere else (?).
I would perhaps call it "the name of the materialized view backing the index".

Reply:
rephrased